src/test/java/com/google/javascript/clutz/DeclarationSyntaxTest.java

```diff
@@ -25,35 +25,24 @@
  */
 @RunWith(JUnit4.class)
 public class DeclarationSyntaxTest {
-  private static final FilenameFilter TS_SOURCES_WITHOUT_PLATFORM_EXTERNS =
+  // types_externs_with_platform has the types from general_with_platform, so it can't be compiled
+  // with everything else
```

Comment:
I feel we are missing the real reason why 'types_externs_with_platform has the type from general_with_platform'.